### PR TITLE
plugins/exit_pulp_publish: tolerate missing annotations

### PR DESCRIPTION
**Pull request by sosiouxme**
Fix for OSBS-5262 in which when a worker build disappears or monitoring
otherwise fails for some reason, pulp_publish gives a traceback when
attempting to find annotations from it.

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -94,3 +94,6 @@ if [[ $OS != "fedora" ]]; then $RUN $PIP install -U setuptools; fi
 
 # Run tests
 $RUN pytest -vv tests --cov atomic_reactor "$@"
+
+echo "To run tests again:"
+echo "$RUN pytest -vv tests --cov atomic_reactor "
```
Comment (mlangsdorf) [inline test.sh]: Seems unrelated to the pulp_publish changes.

**Comment by lcarva** [comment]
Updated release notes draft:
> - `exit_pulp_publish` tolerates missing annotations

File: test.sh (hunk 1 above)
Comment (sosiouxme) [reply test.sh]: It is unrelated... just a convenience for the future
